MapboxDirections.xcodeproj/project.pbxproj

```diff
@@ -632,6 +635,7 @@
 				39D04EFCE1A7300374AA5FBF /* Frameworks */,
 				CA9D333DD4D637ED30DA1B5D /* [CP] Embed Pods Frameworks */,
 				B146922AAE71DC82A56BBA60 /* [CP] Copy Pods Resources */,
+				43F82C8111AB3A1D45F1197F /* Embed Pods Frameworks */,
```

Comment:
That’s weird, there’s a duplicate Embed Pods Frameworks target now. Can you make sure you’re on the latest CocoaPods beta and try again? (The 1.1.0 betas add support for Swift 2.3 and probably account for the [CP] steps above.)